tock/hours/views.py

```diff
@@ -138,12 +138,20 @@ def get_object(self, queryset=None):
     def get_context_data(self, **kwargs):
         context = super(TimecardView, self).get_context_data(**kwargs)
 
-        working_hours = ReportingPeriod.objects.get(
```

Comment:
Right above here, we could probably set another variable, `base_reporting_period`, and set it equal to `ReportingPeriod.objects.get(start_date=self.kwargs['reporting_period'])` to cut down on the number of database calls here.  In fact, we could just set that instead and then reference it down below in lines 152 - 154, e.g.:

```
formset.set_exact_working_hours(base_reporting_period.exact_working_hours)
formset.set_max_working_hours(base_reporting_period.max_working_hours)
formset.set_min_working_hours(base_reporting_period.min_working_hours)
```